### DISCUSSION TITLE
I want to contribute to this project. Please assign me something to do

### DISCUSSION THREAD

**Discussion by innovatorved** [pocketbase/pocketbase#1540] [Q&A]
@ganigeorgiev

**Comment by ganigeorgiev** [comment]
You can refer to https://github.com/pocketbase/pocketbase#contributing.


PocketBase is a hobby project and I want to keep it simple and to avoid unnecessary feature bloating it.

PocketBase has a [roadmap](https://github.com/orgs/pocketbase/projects/2) and PR for new features (especially UI related) are not always helpful because most of the time I don't have a fixed/clear idea how things should be implemented and I usually end up experimenting with different approaches before deciding what will land in `master`. Additionally I work through issues in a specific order and such PRs "forces" a focus change and make me feel obligated to spent time to review and test them, even thought they are not part of the current schedule. I really appreciate the effort and I know that the developer probably has spent several hours just trying to figure out where and how things works, but it would be better to first file a discussion/issue with the planned implementation before investing time to work on new features (unless it is a blocker for your use case and you are OK with maintaining your own fork) because most likely I will not merge it (see for example the recently closed PRs - https://github.com/pocketbase/pocketbase/pull/1503, https://github.com/pocketbase/pocketbase/pull/1507).

If you want to help, you can:

- Report issues/bugs.

- Help with the Apple OAuth2 implementation in https://github.com/pocketbase/pocketbase/pull/899 (I don't have Apple developer account, nor access to Apple device).

- Implement new OAuth2 provider(s).
  For a simple example you can check the recent Strava provider PR - https://github.com/pocketbase/pocketbase/pull/1443 (you can ignore the generated `ui/dist` changes).

- Work on a new SDK (you can use the [Dart SDK](https://github.com/pocketbase/dart-sdk) or [JS SDK](https://github.com/pocketbase/js-sdk) as a reference).

- Suggest docs improvements (typos, grammar errors, better examples, etc.) in https://github.com/pocketbase/site.

**Comment by innovatorved** [comment]
The multiple options are a little confusing to me. Please assign me anything from the roadmap so I can begin working on it. It could be anything, but please assign me something that is beginner-friendly
@ganigeorgiev
